start_databroker_pod.sh

```diff
@@ -6,7 +6,7 @@ IP_ADDR=`ifconfig | sed -En 's/127.0.0.1//;s/.*inet (addr:)?(([0-9]*\.){3}[0-9]*
 
 ########################################################################################################################
 # Dataaccess pod.
-podman pod create -n databroker -p 6942:9090/tcp
+podman pod create -n databroker -p 6942:9090/tcp -p 8888:8888/tcp
```

Comment:
do we want to relocate to a different port?

Reply:
Why not the standard one?

Reply:
Oh, I suppose to avoid collisions with any locally-running Jupyter server. Fair enough. Mapped to `9999`. Updated instructions above to match.